eng/sdl-tsa-vars.config

```diff
@@ -0,0 +1,11 @@
+-SourceToolsList @("policheck","credscan")
+-TsaInstanceURL https://devdiv.visualstudio.com/
+-TsaProjectName DEVDIV
+-TsaNotificationEmail aspnetcore-build@microsoft.com
+-TsaCodebaseAdmin REDMOND\kevinpi
+-TsaBugAreaPath DevDiv\ASP.NET Core
+-TsaIterationPath DevDiv
+-TsaRepositoryName aspnetcore
```

Comment:
@jcagme this & `TsaCodebaseName` weren't in our variable group, do we need them here?

Reply:
No, just add the things you know. If these are required for your repo in particular we'll know when this runs but, if things ran fine and you didn't have these removing them shouldn't hurt

Reply:
Actually, answered before looking at your code. Since you are defining them in ci.yaml then yes, add them to this file as well